plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/OpenLineageListenerConfig.java

```diff
@@ -105,4 +109,31 @@ public OpenLineageListenerConfig setNamespace(String namespace)
         this.namespace = Optional.ofNullable(namespace);
         return this;
     }
+
+    public String getJobNameFormat()
+    {
+        return jobNameFormat.orElse(DEFAULT_JOB_NAME_FORMAT);
+    }
+
+    @Config("openlineage-event-listener.job.nameFormat")
```

Comment:
Please avoid camel case in config names.

Reply:
Ok